package.json

```diff
@@ -42,14 +42,14 @@
     "edition": "octane"
   },
   "devDependencies": {
-    "@appuniversum/ember-appuniversum": "^1.0.1",
+    "@appuniversum/ember-appuniversum": "1.0.1",
```

Comment:
Pinned ember-appuniversum here to `1.0.1` to get it working locally. we might want to wait with merging until a new version is released so this pin can be removed?